conformance/conformance_test.go

```diff
@@ -62,6 +62,7 @@ func TestConformance(t *testing.T) {
 		Client:                     client,
 		RESTClient:                 clientset.CoreV1().RESTClient().(*rest.RESTClient),
 		RestConfig:                 cfg,
+		Clientset:                  clientset,
```

Comment:
Can you add a comment here stating that this is only intended to be used as a stopgap until controller-runtime client supports commands like logs? Maybe link to https://github.com/kubernetes-sigs/controller-runtime/issues/452.